src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -32,8 +32,8 @@ public class IntakeSubsystem extends SubsystemBase {
   private void configStatusFramePeriodsAndBatteryComp(TalonFX talon) {
     talon.setStatusFramePeriod(1, 100);
     talon.setStatusFramePeriod(2, 20);
+    talon.configVoltageCompSaturation(11);
```

Comment:
Why are we reducing the max voltage of the talon?

Reply:
This is the talon saturation value. In essence, it means so long as our battery doesn't go below 11v, we will receive consistent output. The alternative is failing to meet rpm targets or percent targets -- you are always compensating to voltage, but by choosing 11 it means you get consistent shooting for the entire match. See [the bringup tutorial](https://docs.ctre-phoenix.com/en/stable/ch13_MC.html#voltage-compensation) or [the docs](https://api.ctr-electronics.com/phoenix/release/java/com/ctre/phoenix/motorcontrol/can/BaseMotorController.html#configVoltageCompSaturation(double)) for more info.

Reply:
oh ok. That makes sense